test/test-restart-all.js

```diff
@@ -0,0 +1,38 @@
+'use strict';
+
+var _ = require('lodash');
+var cluster = require('cluster');
+var debug = require('debug')('strong-cluster-control:test');
+var master = require('../lib/master');
+var tap = require('tap');
+
+debug('master', process.pid);
+
+cluster.setupMaster({
+  exec: 'test/workers/null.js'
+});
+
+tap.test('restart all workers', function(t) {
+  var SIZE = 5;
+  var DELAY = 200;
+
+  t.plan(4);
```

Comment:
This doesn't look like a test that requires a plan - is the flow less deterministic than it looks? If so, maybe a comment explaining how/why that is?